### PR TITLE
feat: add logging to start runner script.

### PR DESCRIPTION
**Pull request by GuptaNavdeep1983**
To debug the issues around start runner script which is very common during any updates, added a new tag which can be used to print the statements from bash script.

### CODE REVIEW

File: lambdas/functions/control-plane/src/scale-runners/scale-up.ts [hunk 1]
```diff
@@ -205,6 +206,7 @@ export async function createRunners(
     runnerType: githubRunnerConfig.runnerType,
     runnerOwner: githubRunnerConfig.runnerOwner,
     ...ec2RunnerConfig,
+    logLevel: process.env.LOG_LEVEL || 'info',
```
Comment (npalm) [inline lambdas/functions/control-plane/src/scale-runners/scale-up.ts:209]: I think we should stop with reading prcoess.env enverywhre. This is not helping the complexity and understanding wht is happening where. In. this case it is even not needed to read the value from the environment. You can simply request the level in `runner.ts` bij `logger.getLogLevel()`.

File: modules/runners/templates/start-runner.sh [hunk 2]
```diff
@@ -32,6 +32,7 @@ availability_zone=$(curl -f -H "X-aws-ec2-metadata-token: $token" -v http://169.
 environment=$(curl -f -H "X-aws-ec2-metadata-token: $token" -v http://169.254.169.254/latest/meta-data/tags/instance/ghr:environment)
 ssm_config_path=$(curl -f -H "X-aws-ec2-metadata-token: $token" -v http://169.254.169.254/latest/meta-data/tags/instance/ghr:ssm_config_path)
 runner_name_prefix=$(curl -f -H "X-aws-ec2-metadata-token: $token" -v http://169.254.169.254/latest/meta-data/tags/instance/ghr:runner_name_prefix || echo "")
+log_level=$(curl -f -H "X-aws-ec2-metadata-token: $token" -v http://169.254.169.254/latest/meta-data/tags/instance/ghr:log_level || echo "")
```
Comment (npalm) [inline modules/runners/templates/start-runner.sh:35]: The user-data script already read a variable log level. This code should move to the wrapper. And merged with the flag enableUserdataLogging.

File: lambdas/functions/control-plane/src/aws/runners.d.ts [hunk 3]
```diff
@@ -39,4 +39,5 @@ export interface RunnerInputParameters {
   };
   numberOfRunners?: number;
   amiIdSsmParameterName?: string;
+  logLevel?: string;
```
Comment (npalm) [inline lambdas/functions/control-plane/src/aws/runners.d.ts:42]: Not needed, loglevel can be obtained via the logger. See other comments

**Review by npalm — CHANGES_REQUESTED** [review]
The idea looks good to add support to enable activating logging in this way. It gives an option to enable debug logging without rebuilding an AMI. Pleas can you update the PR description with better reasoning why adding this feature?

File: lambdas/functions/control-plane/src/aws/runners.d.ts (hunk 3 above)
Comment (GuptaNavdeep1983) [reply lambdas/functions/control-plane/src/aws/runners.d.ts:42]: removed

**Comment by npalm** [comment]
@GuptaNavdeep1983 what shall we do with this PR?

**Pull request closed without merging**